src/main/java/org/java_websocket/client/WebSocketClient.java

```diff
@@ -226,6 +227,36 @@ public Socket getSocket() {
 		return socket;
 	}
 
+	/**
+	 * Adds an additional header to be sent in the handshake.<br>
+	 * If the connection is already made, adding headers has no effect,
+	 * unless reconnect is called, which then a new handshake is sent.<br>
+	 * If a header with the same key already exists, it is overridden.
+	 */
```

Comment:
Could you please add:

- @since 1.4.1
- some description what key and value represent

here and for the other 2 new methods

Reply:
Thank you for the review. Of course, I am on it right now 😃